src/components/configurable-columns/configurable-columns.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import I18n from 'i18n-js';
+import { DraggableContext } from './../drag-and-drop';
+import Button from './../button';
+import ConfigurableColumnRow from './configurable-column-row';
+import Heading from './../heading';
+import Form from './../form';
+
+class ConfigurableColumns extends React.Component {
+  static propTypes = {
+    /**
+     * Children elements.
+     *
+     * @property children
+     * @type {Node}
+     */
+    children: PropTypes.node,
+
+    /**
+     * A custom class name for the component.
+     *
+     * @property className
+     * @type {String}
+     */
+    className: PropTypes.string,
+
+    /**
+     * Callback triggered when the form is canceled.
+     *
+     * @property onCancel
+     * @type {Function}
+     */
+    onCancel: PropTypes.func.isRequired,
+
+    /**
+     * Callback triggered when an item is dragged.
+     *
+     * @property onDrag
+     * @type {Function}
+     */
+    onDrag: PropTypes.func.isRequired,
+
+    /**
+     * Callback triggered when when the reset button is pressed.
+     *
+     * @property onReset
+     * @type {Function}
+     */
+    onReset: PropTypes.func,
+
+    /**
+     * Callback triggered when the form is saved.
+     *
+     * @property onSave
+     * @type {Function}
+     */
+    onSave: PropTypes.func.isRequired,
+
+    /**
+     * Title to display as heading.
+     *
+     * @property title
+     * @type {String}
+     */
+    title: PropTypes.string.isRequired
```

Comment:
OK that makes sense if the Heading component already treats the `title` prop that way. Will update.